packages/terra-clinical-action-header/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Change dependency on terra-base to v2.0.0
```

Comment:
Maybe not a huge deal, but I believe this update caused all the tests links to receive bullets

Reply:
That's because we're using `<li>` tags and not the List component